### DISCUSSION TITLE
How frequent will packages be updated?

### DISCUSSION THREAD

**Discussion by gertvanantwerpentno** [Entware/Entware#825] [Q&A] [answered]
When I look in the "packages" directory, I see all builts are from 12-Apr-2022. In the mean time, all large Linux-distro's had security updates. How frequent will Entware be updated (in other words, can I be reasonable sure security problems will be fixed as soon as possible)?

**Comment by ryzhovau** [comment, marked as answer]
Thank you to name us as "large" Linux distro:)
We keeping eye on [outdated packages percentage](https://github.com/Entware/Entware/projects/3) to pick appropriate moment.
Usually once per 3 months. 
Every massive update takes ~10 days to run basic tests on fresh packages, then binaries will be rolled out to `bin.entware.net`
We do not track CVEs on real time, it's out of project goal.

**Comment by GwynethLlewelyn** [comment]
Yay! 👏 And we got just another update!
